Rmarkdown/Multiple_Ascending_Dose_PKPD_continuous.Rmd

```diff
@@ -13,70 +13,88 @@ This document contains exploratory plots for continuous PD data as well as the R
 
 ## Setup
 
-```{r, echo = TRUE, warning=FALSE, message=FALSE}
-# remove reference to home directory in libPaths
-.libPaths(grep("home", .libPaths(), value=TRUE, invert=TRUE))
-
-# add localLib to libPaths for locally installed packages
-.libPaths(c("localLib", .libPaths()))
-
-# will load from first filepath first, then look in .libPaths for more packages not in first path
-# version matches package in first filepath, in the case of multiple instances of a package
-
-# library(rmarkdown)
-library(gridExtra)
-library(grid)
+```{r, warning=FALSE, message=FALSE}
 library(ggplot2)
 library(dplyr)
-library(RxODE)
-library(caTools)
-
-theme_set(theme_bw(base_size=12))
+library(tidyr)
+library(xgxr, lib.loc = "../Rlib")
 
 #flag for labeling figures as draft
-  draft.flag      = TRUE
+status = "DRAFT"
 
 ## ggplot settings
-theme_set(theme_bw(base_size=12))
-
-```
-
-
-## Define Useful Graphics Functions
-
-```{r}
-
-# annotation of plots with status of code
-# add draft to plots:draft.flag = TRUE or FALSE - indicates whether to add draft
-AnnotateStatus <- function(draft.flag, x.pos=Inf, y.pos=Inf,
-                           label="DRAFT",color="grey",
-                           hjust=1.2,vjust=1.2,
-                           fontsize=7,alpha = 0.7,
-                           fontface="bold") {
-  
-    if(draft.flag) {
-    annotateStatus <- annotate("text",x=x.pos,y=y.pos,
-             label=label,
-             color=color,
-             hjust=hjust,vjust=vjust,
-             cex=fontsize, alpha = alpha,fontface=fontface)
-    } else {
-    annotateStatus <- NULL
-  }
-    return(annotateStatus)
-  }  
-
+xgx_theme_set()
 
 ```
 
 ## Load Dataset
 
 ```{r, warning=FALSE, message=FALSE}
-my.data <- read.csv("../Data/Multiple_Ascending_Dose_Dataset2.csv")
-
-# Define order for factors
-my.data$TRTACT <- factor(my.data$TRTACT, levels = unique(my.data$TRTACT[order(my.data$DOSE)]))
 
+#load dataset
+pkpd_data <- read.csv("../Data/Multiple_Ascending_Dose_Dataset2.csv")
+
+#ensure dataset has all the necessary columns
+  pkpd_data = pkpd_data %>%
+    mutate(ID      = ID,     #ID   column
+           TIME    = TIME,   #TIME column name 
+           NOMTIME = NOMTIME,#NOMINAL TIME column name
+           EVID    = EVID   ,#EVENT ID, >=1 is dose, otherwise measurement
```

Comment:
PROFDAY = PROFDAY, #PROFILE DAY day associated with profile, e.g. day of dose administration